torchaudio/__init__.py

```diff
@@ -11,7 +11,12 @@
     transforms,
     utils,
 )
-from torchaudio.backend import get_audio_backend, list_audio_backends, set_audio_backend
+
+import os
+if os.getenv("TORCHAUDIO_USE_BACKEND_DISPATCHER") == "1":
+    import torchaudio._backend  # noqa: F401
+else:
+    from torchaudio.backend import get_audio_backend, list_audio_backends, set_audio_backend
```

Comment:
I think it's better to keep these attributes available and change only the behavior when `TORCHAUDIO_USE_BACKEND_DISPATCHER` is enabled, like warning message and no-op.